class/abstract-hd-electrum-wallet.js

```diff
@@ -1,7 +1,8 @@
-import { NativeModules } from 'react-native';
 import bip39 from 'bip39';
 import BigNumber from 'bignumber.js';
 import b58 from 'bs58check';
+import crypto from 'crypto';
+import { promisify } from 'util';
```

Comment:
Maybe something like:

```suggestion
import { promisify } from 'util';
const randomBytes = promisify(crypto.randomBytes);
```

Near the top, and instead of `await crypto.randomBytes(32)` just do `await randomBytes(32)`